keylime/ima/ast.py

```diff
@@ -374,7 +374,9 @@ def __init__(
             raise ParserError(f"No parser for mode {tokens[2]} implemented.")
         self.mode = mode(tokens[3])
         self._bytes = self.mode.bytes()
-        self.pcr_template_hash = self._pcr_hash_alg.hash(self._bytes)
+        pcr_hash_alg_ = self._pcr_hash_alg.hash(self._bytes)
+        assert pcr_hash_alg_ is not None
```

Comment:
Likely resolved with PR #1189 -- at least I don't see it anymore when running `pyright keylime/ima/ast.py`.

The good part is probably the change above to get rid of Optional.

Reply:
I dropped the assert.